internal/core/graphics/color.rs

```diff
@@ -192,6 +194,52 @@ impl Color {
         let rgba: RgbaColor<f32> = hsva.into();
         rgba.into()
     }
+
+    /// Returns a new version of this color with the opacity changed by _subtracting_
+    /// `amount` from it. The reference is the opacity's normalized value as `f32`.
+    ///
+    /// This means that a _positive_ `amount` will yield a more transparent color and
+    /// a _negative_ `amount` will yield a more opaque color.
+    #[must_use]
+    pub fn faded(&self, amount: f32) -> Self {
+        let mut rgba: RgbaColor<f32> = (*self).into();
+        rgba.alpha = rgba.alpha.sub(amount).clamp(0.0, 1.0);
+        rgba.into()
+    }
+
+    /// Returns a new color that is a mix of `self` and `other`, with a proportion
+    /// factor given by `factor` (which should be between `0.0` and `1.0`).
+    ///
+    /// This will make a weighted average of the corresponding RGBA channels,
+    /// with weight `factor` for `self` and `1 - factor` for `other`, and produce
+    /// the color with the resulting values.
+    #[must_use]
+    pub fn mixed(&self, other: &Self, factor: f32) -> Self {
```

Comment:
Is `interpolated` or `lerp` not the term?
I believe we should use this function from `impl InterpolatedPropertyValue for Color` as well (or the other way around)

Reply:
Yes, I thought about using `lerp`, but as this is more of a "linear algebra" term and I was mimicking a function from SASS with the same name I chose `mixed` instead.

Do you think I should change? I don't mind doing it, but I would also vote for consistency...

Reply:
> I believe we should use this function from `impl InterpolatedPropertyValue for Color` as well (or the other way around)

I'm not familiar with the codebase yet, so I'm not sure I understood.

Did you mean there is already an implementation for this in that trait or that this one function could be used for that? (Or maybe both, now that I read it again...?)

Reply:
ok for `mixed`.  CSS itself has `color-mix`
 But transparentize then would also be consistant with sass